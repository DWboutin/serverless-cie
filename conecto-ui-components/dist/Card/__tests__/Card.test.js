"use strict";

var _interopRequireDefault = require("@babel/runtime/helpers/interopRequireDefault");

var _react = _interopRequireDefault(require("react"));

var _enzyme = require("enzyme");

var _Card = _interopRequireDefault(require("../../Card"));

var _Icon = _interopRequireDefault(require("../../Icon"));

describe('Card', () => {
  it('should match snapshot', () => {
    const wrapper = (0, _enzyme.shallow)(_react.default.createElement(_Card.default, null, "Test 1"));
    expect(wrapper).toMatchSnapshot();
  });
  it('should match snapshot with an icon', () => {
    const wrapper = (0, _enzyme.shallow)(_react.default.createElement(_Card.default, {
      withIcon: true,
      icon: _react.default.createElement(_Icon.default, {
        type: "colored-flag"
      })
    }, "Test 1"));
    expect(wrapper).toMatchSnapshot();
  });
  it('should have the prefix class', () => {
    const wrapper = (0, _enzyme.shallow)(_react.default.createElement(_Card.default, null, "Test 1"));
    expect(wrapper.hasClass('to-card')).toEqual(true);
  });
});