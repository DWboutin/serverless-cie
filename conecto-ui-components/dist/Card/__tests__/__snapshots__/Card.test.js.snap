// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card should match snapshot 1`] = `
<div
  className="to-card to-card-bordered"
>
  <div
    className="to-card-wrap"
  >
    Test 1
  </div>
</div>
`;

exports[`Card should match snapshot with an icon 1`] = `
<div
  className="to-card to-card-bordered to-card-with-icon"
>
  <div
    className="icon-wrapper"
  >
    <Icon
      type="colored-flag"
    />
  </div>
  <div
    className="to-card-wrap"
  >
    Test 1
  </div>
</div>
`;
